### PR TITLE
Incident Management and Security Signals

### PR DESCRIPTION
**Pull request by amberben**
Described how to declare incidents for Security Signals and linked to Incidents page

### What does this PR do?
Introduce new incident management capability for security signals (ASM, CWS and Cloud SIEM)

### Motivation
Doing a beta program for Incidents but wanted to add documentation for customers curious about the how and why of the functionality

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by maycmlee — CHANGES_REQUESTED** [review]
Hey @amberben, left some suggestions!

File: content/en/security_platform/security_signal_management/_index.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ To change signal state, follow the instructions below:
 
 4. Once you save the status, a confirmation toast appears that states which action was taken with the opportunity to “Undo” the action. You'll see it in a banner above the signal side panel when you save the action. The banner shows the action taken, by who, and when.
 
+Any security signal that may lead to a disruption in your organization’s services can be described as an incident, and it is often necessary to have a set framework for handling these threats. [Incident Management][6] provides a system through which your organization can effectively identify and mitigate incidents.
```
[the remark this answers is not in the document]
Comment (amberben) [reply content/en/security_platform/security_signal_management/_index.md:51]: lgtm, thank you!

File: content/en/security_platform/security_signal_management/_index.md [hunk 2]
```diff
@@ -48,6 +48,12 @@ To change signal state, follow the instructions below:
...
+
+You can declare an incident directly from a Cloud SIEM or Cloud Workload Security signal by clicking the kebab button in the top-right of the side panel, or by selecting the export button in the Application Security Monitoring side panel, and clicking **Declare incident**. The incident creation modal appears, and the security signal is added to the incident as a signal.
```
[the remark this answers is not in the document]
Comment (amberben) [reply content/en/security_platform/security_signal_management/_index.md:53]: Fantastic! Appreciate the review :)

**Comment by amberben** [comment]
@maycmlee totally ok with merging this!

**Review by maycmlee — APPROVED** [review]

**Pull request merged**